libraries/rush-lib/src/logic/operations/CacheableOperationPlugin.ts

```diff
@@ -429,16 +433,12 @@ export class CacheableOperationPlugin implements IPhasedCommandPlugin {
         }
 
         // No need to run for the following operation status
-        switch (record.status) {
-          case OperationStatus.NoOp:
-          case OperationStatus.RemoteExecuting: {
-            return;
-          }
-          default: {
-            break;
-          }
+        if (!record.isTerminal || record.status === OperationStatus.NoOp) {
+          return;
         }
 
+        this._itemsToCheck.delete(record);
```

Comment:
This line seems to be the only reference to this property; delete?

Reply:
Leftover from another attempt. Removed in [0a86ddc](https://github.com/microsoft/rushstack/pull/4821/commits/0a86ddc0e43f233564ad23ac92f4e3c78c1a8a3f)